dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -1162,3 +1229,103 @@ void CommandBarFlyoutCommandBar::ClearShadow()
+                            {
+                                actualFlyout->AddDropShadow();
+                            }
+                        }
+                    }
+                }
+            });
+    }
+
+    m_expandedUpToCollapsedStoryboard.set(GetTemplateChildT<winrt::Storyboard>(L"ExpandedUpToCollapsed", thisAsControlProtected));
+    if (m_expandedUpToCollapsedStoryboard)
+    {
+        m_expandedUpToCollapsedStoryboardRevoker = m_expandedUpToCollapsedStoryboard.get().Completed(winrt::auto_revoke,
+            {
+                [this](auto const&, auto const&)
+                {
+                    if (SharedHelpers::IsAnimationsEnabled())
+                    {
+                        if (auto owningFlyout = m_owningFlyout.get())
+                        {
+                            if (auto actualFlyout = winrt::get_self<CommandBarFlyout>(owningFlyout))
+                            {
+                                actualFlyout->AddDropShadow();
+                            }
+                        }
+                    }
+                }
+            });
+    }
+
+    m_collapsedToExpandedUpStoryboard.set(GetTemplateChildT<winrt::Storyboard>(L"CollapsedToExpandedUp", thisAsControlProtected));
+    if (m_collapsedToExpandedUpStoryboard)
+    {
+        m_collapsedToExpandedUpStoryboardRevoker = m_collapsedToExpandedUpStoryboard.get().Completed(winrt::auto_revoke,
+            {
+                [this](auto const&, auto const&)
+                {
+                    if (SharedHelpers::IsAnimationsEnabled())
+                    {
+                        if (auto owningFlyout = m_owningFlyout.get())
+                        {
+                            if (auto actualFlyout = winrt::get_self<CommandBarFlyout>(owningFlyout))
+                            {
+                                actualFlyout->AddDropShadow();
+                            }
+                        }
+                    }
+                }
+            });
+    }
+
+    m_collapsedToExpandedDownStoryboard.set(GetTemplateChildT<winrt::Storyboard>(L"CollapsedToExpandedDown", thisAsControlProtected));
+    if (m_collapsedToExpandedDownStoryboard)
+    {
+        m_collapsedToExpandedDownStoryboardRevoker = m_collapsedToExpandedDownStoryboard.get().Completed(winrt::auto_revoke,
+            {
+                [this](auto const&, auto const&)
+                {
+                    if (SharedHelpers::IsAnimationsEnabled())
+                    {
+                        if (auto owningFlyout = m_owningFlyout.get())
+                        {
+                            if (auto actualFlyout = winrt::get_self<CommandBarFlyout>(owningFlyout))
+                            {
+                                actualFlyout->AddDropShadow();
+                            }
+                        }
+                    }
+                }
+            });
+    }
```

Comment:
This Completed() handler is the same for all the used storyboards here. You don't need to repeat the code 4 times :)
You might want to either extract it as a private function or create a lambda function in the beginning of the function and then reuse it?:) Not 100% sure how to do that:) Lmk if you have problems with this, I can help figure it out